lib/stats.js

```diff
@@ -12,14 +13,15 @@ var platformToMethod = {
   aix: 'ps'
 }
 
+var useProcfile = fs.existsSync('/etc/alpine-release')
```

Comment:
@simonepri I had to merge real quick but we can still discuss this. I'm not sure we need to add a new dependency for this (I'd like to avoid it). Also `getos` is nice but calls a lot of `fs.stat` to get back results where I only want to know if this is alpine or not. 

The docker test works really well, if you have other OSes in mind that we could test like this please let me know!